tests/components/zwave_js/test_device_trigger.py

```diff
@@ -252,7 +252,13 @@ async def test_if_entry_control_notification_fires(
             "event": "notification",
             "nodeId": node.node_id,
             "ccId": 111,
-            "args": {"eventType": 5, "dataType": 2, "eventData": "555"},
+            "args": {
```

Comment:
This change looks like it belongs in the other PR.

Reply:
yup whoops 🤦🏾  will fix both later